packages/client/__tests__/MtvRoomUsersSearch.test.tsx

```diff
@@ -396,7 +393,7 @@ const mtvRoomUsersSearchMachine = mtvRoomUsersSearchModel.createMachine({
                             }: MtvRoomUsersSearchMachineState,
                         ) => {
                             const filteredUsers = filterUsersByNickname(
-                                fakeUsers,
+                                fakeFollowingAndUsers,
```

Comment:
Please note the distinction with previous code. Following users are part of overall users list. Previously there were two independent lists: friends and users.

Reply:
I can see that you do test that both fakeFollowing and fakeUsers can occurs on a user search result, but in reality I'm not sure that this would work in the case of a search query matching a following nickname :thinking: